### ISSUE TITLE
Nuxt fails after installing @nuxtjs/storybook

### ISSUE THREAD

**Issue by agantelin** [nuxt-modules/storybook#689]
After installing @nuxtjs/storybook to an existing nuxt project, the command `npm run dev` fails with the error:

```
 ERROR  Error while requiring module @nuxtjs/storybook: Error: Cannot find module 'C:\p\project_name\node_modules\@nuxtjs\storybook\dist\module.cjs'                13:39:12  


 ERROR  Cannot start nuxt:  Cannot find module 'C:\p\project_name\node_modules\@nuxtjs\storybook\dist\module.cjs'                                                   13:39:12  

  at createEsmNotFoundErr (node:internal/modules/cjs/loader:1182:15)
  at finalizeEsmResolution (node:internal/modules/cjs/loader:1170:15)
  at resolveExports (node:internal/modules/cjs/loader:592:14)
  at Module._findPath (node:internal/modules/cjs/loader:669:31)
  at Module._resolveFilename (node:internal/modules/cjs/loader:1131:27)
  at Function.resolve (node:internal/modules/helpers:190:19)
  at Function._resolve [as resolve] (node_modules\jiti\dist\jiti.js:1:241814)
  at resolveModule (/C:/p/project_name/node_modules/@nuxt/kit/dist/index.mjs:2224:29)
  at requireModule (/C:/p/project_name/node_modules/@nuxt/kit/dist/index.mjs:2229:24)
  at loadNuxtModuleInstance (/C:/p/project_name/node_modules/@nuxt/kit/dist/index.mjs:2508:92)
```

Directory \node_modules\@nuxtjs\storybook\ contains only node_modules dir & package.json file.
executing `npm run build` in this dir results to the error:

```
> @nuxtjs/storybook@8.1.2 build
> nuxt-module-build build

ℹ Building storybook                                                                                                                                                              13:46:40
ℹ Cleaning dist directory: ./dist                                                                                                                                                 13:46:40

 ERROR  Could not resolve entry module "src/module".                                                                                                                               13:46:40

  at error (/C:/p/project_name/node_modules/@nuxtjs/storybook/node_modules/unbuild/node_modules/rollup/dist/es/shared/node-entry.js:2287:30)
  at ModuleLoader.loadEntryModule (/C:/p/project_name/node_modules/@nuxtjs/storybook/node_modules/unbuild/node_modules/rollup/dist/es/shared/node-entry.js:24881:20)
  at async Promise.all (index 0)


 ERROR  Could not resolve entry module "src/module".   
```

Windows 11
node 20.15
npm 10.8.1
Nuxt 3.12.3 with Nitro 2.9.7
@nuxtjs/storybook 8.1.2

**Comment by 321zeno** [comment]
Did you run `npx storybook-nuxt init`?

**Comment by Kreezag** [comment]
The same issue
Mac Pro M1
node 22.4.1
yarn 1.12

nuxt: 3.12.3
@nuxtjs/storybook 8.1.2

Node throws an error on any command run (package installation, build, generate, running `npx storybook-nuxt init`)

**Comment by tobiasdiez** [comment]
Yes, for some reason the released package is completely empty: https://www.npmjs.com/package/@nuxtjs/storybook?activeTab=code

@chakAs3 could you please try it again? Please don't change the lock file locally (it broke ci again and may be the reason for the faulty release). If you still have these version issues, I'll try to resolve them before you release the next version.

**Comment by chakAs3** [comment]
@tobiasdiez The lockfile changes every time I run pnpm i, even though I haven't made any changes. This prevents the release from passing because the git status isn't clean. I will clone the repository again to get a fresh copy. and see what happens

**Comment by tobiasdiez** [comment]
Do you have an up to date pnpm?

**Comment by chakAs3** [comment]
> Do you have an up to date pnpm?

that is what i doubt i guess i have 8.5.0, should it be 9.x ?

**Comment by tobiasdiez** [comment]
Yes!  The easiest way is to use `corepack enable pnpm` which then will always use the Pnpm version specified in the package json

**Comment by chakAs3** [comment]
@tobiasdiez i have updated pnpm, lockfile is fine not touched, i ran `pnpm release` packages are published.

**Comment by chakAs3** [comment]
same issue the package is empty and no release pushed.

**Comment by tobiasdiez** [comment]
I think I've found the mistake. Hopefully fixed by #702.
